multiuser/keycloak/che-multiuser-keycloak-server/src/main/java/org/eclipse/che/multiuser/keycloak/server/KeycloakEnvironmentInitalizationFilter.java

```diff
@@ -103,6 +106,8 @@ public void doFilter(ServletRequest request, ServletResponse response, FilterCha
       }
     }
 
+    putIdentityIdToMDC(subject);
```

Comment:
I think you can replace it with just ```MDC.put(IDENTITY_ID_MDC_KEY, subject.getUserId());```

Reply:
wait. It should not be here. 
I propose to make a separate filter in core/commons/che-core-commons-logback/ that will put subject id in mdc.